### ISSUE TITLE
Webpack error

### ISSUE THREAD

**Issue by iuliux** [fedoryakubovich/react-awesome-stars-rating#48]
When running `npm start`, errors below pop up with the latest version.
Before upgrading "react-awesome-stars-rating" (`0.1.9`) everything was fine.

Versions:

- react-awesome-stars-rating: 0.2.0
- @babel/core: 7.8.4
- webpack: 4.41.6
- webpack-dev-server: 3.10.3
- npm: 6.13.4

```
ERROR in ../node_modules/react-awesome-stars-rating/dist/index.js
Module not found: Error: Can't resolve '/Users/fedoryakubovich/Documents/projects/react-awesome-stars-rating/node_modules/@babel/runtime/helpers/esm/assertThisInitialized' in '/Users/iuliux/platty/portal/static/js/node_modules/react-awesome-stars-rating/dist'
 @ ../node_modules/react-awesome-stars-rating/dist/index.js 6:0-168 27:47-69 28:49-71 29:41-63 30:43-65 31:37-59
 @ ./index.js
 @ multi webpack-dev-server/client?http://0.0.0.0:8000/ webpack/hot/only-dev-server react-hot-loader/patch ./

ERROR in ../node_modules/react-awesome-stars-rating/dist/index.js
Module not found: Error: Can't resolve '/Users/fedoryakubovich/Documents/projects/react-awesome-stars-rating/node_modules/@babel/runtime/helpers/esm/classCallCheck' in '/Users/iuliux/platty/portal/static/js/node_modules/react-awesome-stars-rating/dist'
 @ ../node_modules/react-awesome-stars-rating/dist/index.js 2:0-154 20:4-19
 @ ./index.js
 @ multi webpack-dev-server/client?http://0.0.0.0:8000/ webpack/hot/only-dev-server react-hot-loader/patch ./

ERROR in ../node_modules/react-awesome-stars-rating/dist/index.js
Module not found: Error: Can't resolve '/Users/fedoryakubovich/Documents/projects/react-awesome-stars-rating/node_modules/@babel/runtime/helpers/esm/createClass' in '/Users/iuliux/platty/portal/static/js/node_modules/react-awesome-stars-rating/dist'
 @ ../node_modules/react-awesome-stars-rating/dist/index.js 3:0-148 35:2-14
 @ ./index.js
 @ multi webpack-dev-server/client?http://0.0.0.0:8000/ webpack/hot/only-dev-server react-hot-loader/patch ./

ERROR in ../node_modules/react-awesome-stars-rating/dist/index.js
Module not found: Error: Can't resolve '/Users/fedoryakubovich/Documents/projects/react-awesome-stars-rating/node_modules/@babel/runtime/helpers/esm/getPrototypeOf' in '/Users/iuliux/platty/portal/static/js/node_modules/react-awesome-stars-rating/dist'
 @ ../node_modules/react-awesome-stars-rating/dist/index.js 5:0-154 22:45-60
 @ ./index.js
 @ multi webpack-dev-server/client?http://0.0.0.0:8000/ webpack/hot/only-dev-server react-hot-loader/patch ./

ERROR in ../node_modules/react-awesome-stars-rating/dist/index.js
Module not found: Error: Can't resolve '/Users/fedoryakubovich/Documents/projects/react-awesome-stars-rating/node_modules/@babel/runtime/helpers/esm/inherits' in '/Users/iuliux/platty/portal/static/js/node_modules/react-awesome-stars-rating/dist'
 @ ../node_modules/react-awesome-stars-rating/dist/index.js 7:0-142 15:2-11
 @ ./index.js
 @ multi webpack-dev-server/client?http://0.0.0.0:8000/ webpack/hot/only-dev-server react-hot-loader/patch ./

ERROR in ../node_modules/react-awesome-stars-rating/dist/index.js
Module not found: Error: Can't resolve '/Users/fedoryakubovich/Documents/projects/react-awesome-stars-rating/node_modules/@babel/runtime/helpers/esm/objectSpread2' in '/Users/iuliux/platty/portal/static/js/node_modules/react-awesome-stars-rating/dist'
 @ ../node_modules/react-awesome-stars-rating/dist/index.js 1:0-151 178:16-29
 @ ./index.js
 @ multi webpack-dev-server/client?http://0.0.0.0:8000/ webpack/hot/only-dev-server react-hot-loader/patch ./

ERROR in ../node_modules/react-awesome-stars-rating/dist/index.js
Module not found: Error: Can't resolve '/Users/fedoryakubovich/Documents/projects/react-awesome-stars-rating/node_modules/@babel/runtime/helpers/esm/possibleConstructorReturn' in '/Users/iuliux/platty/portal/static/js/node_modules/react-awesome-stars-rating/dist'
 @ ../node_modules/react-awesome-stars-rating/dist/index.js 4:0-176 22:12-38
 @ ./index.js
 @ multi webpack-dev-server/client?http://0.0.0.0:8000/ webpack/hot/only-dev-server react-hot-loader/patch ./
```

**Comment by fedoryakubovich** [comment]
@iuliux you need to update the package to the **latest version** (**v0.2.3**).

 The issue was fixed in #49 . If you will face with this problem again please let me know.

**Comment by iuliux** [comment]
@fedoryakubovich Thanks for the quick fix
